setup.py

```diff
@@ -170,6 +170,11 @@
     "unidic_lite>=1.0.7",
     "uvicorn",
     "beautifulsoup4",
+    "sudachipy>=0.6.6",
+    "sudachidict_core>=20220729",
+    "sudachidict_small>=20220729",
+    "sudachidict_full>=20220729",
+    "pyknp>=0.6.1",
```

Comment:
Do we need all five of those for the test?

Reply:
`sudachipy`, `sudachidict_core`, `pyknp` are required.
I added `sudachidict_small` and `sudachidict_full` to test sudachi_dict_type option, but since this is a sudachi feature, it may not have been necessary to add those tests to transformers.
Would it be preferable to remove `sudachidict_small` and `sudachidict_full`, and the corresponding tests?

Reply:
If you don't mind, yes!